src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -84,11 +84,11 @@ def load_arguments(self, _):
 
     with self.argument_context('acr build') as c:
         c.argument('build_id', help='The unique build identifier.')
-        c.argument('image_name', options_list=['--tag', '-t'], help="The image repository and optionally a tag in the 'repository:tag' format.")
+        c.argument('image_name', options_list=['--image', '-t'], help="The image repository and optionally a tag in the 'repository:tag' format.")
```

Comment:
I thought you mentioned docker's convention is to use `--tag` and `-t`?

Reply:
Yes. -t is used in most docker build commands as we can see. As you mentioned, --tag is already used in existing acr commands as a different definition. So I think it's a good tradeoff here.